docs/api_v0.yml

```diff
@@ -2427,6 +2484,76 @@ paths:
               -d '{"classified_listing":{"title":"Title"}' \
               https://dev.to/api/listings/{id}
 
+  /podcast_episodes:
+    get:
+      operationId: getPodcastEpisodes
+      summary: Published podcast episodes
+      description: |
+        This endpoint allows the client to retrieve a list of podcast episodes.
+
+        "Podcast episodes" are episodes belonging to podcasts.
+
+        It will only return active podcast episodes that belong to published
+        podcasts available on the platform,
+        ordered by descending publication date.
+
+        It supports pagination, each page will contain `30` articles by default.
+      tags:
+        - podcast-episodes
+      parameters:
+        - name: page
+          in: query
+          description: Pagination page.
+          schema:
+            type: integer
+            format: int32
+          example: 1
+        - name: per_page
+          in: query
+          description: Page size (defaults to 30 with a maximum of 1000).
+          schema:
+            type: integer
+            format: int32
+          example: 30
+        - name: username
+          in: query
+          description: |
+            Using this parameter will retrieve episodes belonging
+            to a specific podcast.
+          schema:
+            type: string
+          example: codenewbie
+      responses:
+        "200":
+          description: A list of podcast episodes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/PodcastEpisode"
+              examples:
+                articles-success:
+                  $ref: "#/components/examples/PodcastEpisodes"
+        "404":
+          description: Resource not found
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/APIError"
+              examples:
+                article-not-found:
+                  $ref: "#/components/examples/ErrorNotFound"
+      x-code-samples: # https://github.com/Redocly/redoc/blob/master/docs/redoc-vendor-extensions.md#x-code-samples
+        - lang: Shell
+          label: curl (all podcast episodes)
+          source: |
+            curl https://dev.to/api/podcast_episodes
+        - lang: Shell
+          label: curl (all episodes belonging of a podcast)
```

Comment:
Again, not a blocker, but I think it might make more sense to word this line like: "all episodes belonging to a podcast" rather than "...of a podcast". 😄

Reply:
great catch! I'll fix it in the next PR! there are a few coming down the pipeline :D